modules/network/vpc/main.tf

```diff
@@ -65,64 +117,13 @@ module "vpc" {
   project_id                             = var.project_id
   auto_create_subnetworks                = false
   subnets                                = local.all_subnets
+  secondary_ranges                       = var.secondary_ranges
   routing_mode                           = var.network_routing_mode
   mtu                                    = var.mtu
   description                            = var.network_description
   shared_vpc_host                        = var.shared_vpc_host
   delete_default_internet_gateway_routes = var.delete_default_internet_gateway_routes
-}
-
-module "firewall_rules" {
```

Comment:
What is the reason behind no longer using this module? Advantages & disadvantages?

Reply:
I have chosen to revert the migration because it doesn't cleanly handle create/destroy for existing infra. We should wait until we are willing to impose Terraform 1.1 (or Terraform can silently ignore blocks it doesn't yet understand) to do this